dev/linters/shellcheck.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+RED='\033[0;31m'
+NC='\033[0m'
```

Comment:
Why do we have a completely different style of colourizing text here?

Reply:
Historical